boards/arm/nucleo_h745zi_q/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 if(CONFIG_PINMUX)
-zephyr_library()
-zephyr_library_sources(pinmux.c)
 zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
 endif()
```

Comment:
If aim is to remove pinmux , can't we get ride of 
```
if(CONFIG_PINMUX) 
```
?

Reply:
I can confirm, that removing of above `if` in `CMakeLists.txt` works also fine and is possible.